listeners.go

```diff
@@ -392,11 +392,11 @@ func SplitNetworkAddress(a string) (network, host, port string, err error) {
 	}
 	if IsUnixNetwork(network) {
 		host = a
-		return
+		return network, host, port, nil
```

Comment:
This seems like a mistake -- the returns are named.

Reply:
ahhh, so it isn't a mistake actually, it's designed to make the code clearer.  If you'd like me to change it back the way it was, just let me know.